ui/imports/Utils.qml

```diff
@@ -197,4 +197,31 @@ QtObject {
             default: return network
         }
     }
+
+    function validate(item, firstPasswordField, repeatPasswordField) {
```

Comment:
Can we rename this to `validatePasswords` or something like that

Reply:
Done!